.github/workflows/dev-cd.yaml

```
@@ -8,6 +8,7 @@ env:
   REACT_APP_ETHERSCAN_KEY: ${{ secrets.REACT_APP_ETHERSCAN_KEY }}
   REACT_APP_INFURA_ID: ${{ secrets.REACT_APP_INFURA_ID }}
   REACT_APP_MULTI_SEND_CONTRACT: "0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761"
+  REACT_APP_BACKEND_API_URL: "http://api.dev.zodiac.gnosisguild.org/api"
```

```suggestion
  REACT_APP_BACKEND_API_URL: "https://api.dev.zodiac.gnosisguild.org/api"
```
Can't we use https? 🤔

---

whoops, yes we can and should 🥂